deploy/templates/default-azuredeploy-webjobs.json

```diff
@@ -197,11 +197,20 @@
       "metadata": {
         "description": "The domain of the Azure Storage Account."
       }
+    },
+    "EventHubsDomain": {
+      "type": "string",
+      "defaultValue": "servicebus.windows.net",
+      "metadata": {
+        "description": "The domain of the Azure Event Hubs."
+      }
     }
+
   },
   "variables": {
     "asa_job_name": "[parameters('ServiceName')]",
     "eventhub_namespace_name": "[parameters('ServiceName')]",
+    "eventhub_fqdn": "[concat(variables('eventhub_namespace_name'), '.', parameters('EventHubsDomain'))]",
```

Comment:
The below should work as a cross cloud solution for obtaining the FQDN of the EventHub:

```
[reference(resourceId('Microsoft.EventHub/namespaces', variables('eventhub_namespace_name'))).serviceBusEndpoint]
```

Reply:
I tried this out but it seems to only work for resources that are "outputted" or already created.

For inputs it seemed to return this error: "The template function 'reference' is not expected at this location.". Will probably just leave as is for now.